web/packages/teleport/src/ironrdp/src/lib.rs

```diff
@@ -27,16 +31,12 @@ use ironrdp_session::{
     fast_path::Processor as IronRdpFastPathProcessor,
     fast_path::ProcessorBuilder as IronRdpFastPathProcessorBuilder,
 };
+use ironrdp_web::image::extract_partial_image;
```

Comment:
Will this still be possible based on discussion in https://github.com/Devolutions/IronRDP/pull/449?

Reply:
No I'm going to have to revert this.